docs/semgrep-app/sso.md

```diff
@@ -61,4 +61,21 @@ To set up SSO:
 
 If you encounter issues during the setup process, reach out to [support@r2c.dev](mailto:support@r2c.dev) for assistance.
 
-<MoreHelp />
+### Setting up SAML SSO with Azure Active Directory
+
+Set up Semgrep as an Enterprise App with SAML SSO using Azure's Active Directory service by following these steps:
+
+1. Enable **Enterprise App** in **Azure Active Directory** with SAML SSO selected. Follow the instructions in [Enable single sign-on for an enterprise application](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso).
+2. Go to Semgrep App [Settings](https://semgrep.dev/orgs/-/settings/sso) page. TODO Under **Basic Configuration**, you'll need to enter the `Entity ID` and the `Reply URL`. These are provided in the Semgrep App on the left-side **Settings** tab, under the **SSO** tab on the top, as `Audience URL` and `SSO URL`, respectively. Make sure you switch the provider type to **SAML2** in the SSO form in the Semgrep App.
```

Comment:
I don't see **Basic Configuration** in the Semgrep App. Do we mean **Basic SAML Configuration** on the side of Azure? We need to specify at the beginning of each step where we are. These steps are too long, there should be one or two actions at maximum for a user to do in each step.